src/content/guide/how-to-build-a-product/manufacturing.md

```diff
@@ -15,7 +15,20 @@ In most cases, manufacturing with Particle hardware is the same as manufacturing
 ### Purchasing Particle hardware at volume
 You have two options when it comes to purchasing Particle hardware--purchasing from Particle, and purchasing directly from our module manufacturer. Let's run through the pros and cons of each:
 
-- **Purchasing from Particle** means that you're buying either Photons or PØ/P1 modules through our [online store](http://store.particle.io), or purchasing larger volumes by [contacting sales](http://www.particle.io/sales).  They'll be shipped to you from our warehouses in cut tape strips of ten (10) units, or as complete reels. There are several advantages to purchasing directly from Particle. In general, we try to keep hardware in stock so that we can provide quick lead times and prompt delivery timelines. We've already established relationships with our module manufacturer, so you can leverage our order volumes to get better pricing. Working with large silicon manufacturers can be difficult and slow, and we strive to be friendly, communicative, and easy to deal with. Additionally, when you buy hardware from Particle, it includes all the necessary firmware libraries and system firmware for connecting to the Particle Cloud by default, simplifying your manufacturing process.  This option is best for low to medium volume product deployments (<50K units).
+- **Purchasing from Particle** means that you're buying either Photons
+or PØ/P1 modules through our [online store](http://store.particle.io),
```

Comment:
Why do yo wrap lines in some places at 70 characters?

Reply:
My editor's default for .md files :)

Reply:
@jme783 https://github.com/particle-iot/docs/pull/779 😉